roles/tendrl-server/README.md

```diff
@@ -32,6 +41,12 @@ Role Variables
   port for `tendrl-node-agent` will be skipped so that the default value from
   config file (as shipped in rpm package) will be used. *If you are not sure*
   if you need to reconfigure this, *leave this variable undefined*.
+* When `etcd_authentication` variable is undefined or set to `True` (which is
+  the default value), this role will configure [etcd
+  authentication](https://coreos.com/etcd/docs/latest/op-guide/authentication.html).
+  When the value is `False`, the authentication will be disabled, which is
+  useful for development/testing purposes only. For production, keep the
+  authentication always enabled.
```

Comment:
If I understand it well, following sentence is not true:
> When the value is `False`, the authentication will be disabled...
because when authentication in etcd is already enabled, it will not disable it.

Reply:
Good point.

Reply:
Addressed in 39210f7847f53e8eb4e02ba60a581af860fe6e53